src/pymanopt/manifolds/positive_definite.py

```diff
@@ -49,34 +28,33 @@ def dist(self, point_a, point_b):
         c = np.linalg.cholesky(point_a)
         c_inv = np.linalg.inv(c)
         logm = multilogm(
-            c_inv @ point_b @ multitransp(c_inv),
+            c_inv @ point_b @ multihconj(c_inv),
             positive_definite=True,
         )
-        return np.linalg.norm(logm)
+        return np.real(np.linalg.norm(logm))
 
     def inner_product(self, point, tangent_vector_a, tangent_vector_b):
-        p_inv_tv_a = np.linalg.solve(point, tangent_vector_a)
-        if tangent_vector_a is tangent_vector_b:
-            p_inv_tv_b = p_inv_tv_a
-        else:
-            p_inv_tv_b = np.linalg.solve(point, tangent_vector_b)
-        return np.tensordot(
-            p_inv_tv_a, multitransp(p_inv_tv_b), axes=tangent_vector_a.ndim
+        return np.real(
+            np.tensordot(
+                np.linalg.solve(point, tangent_vector_a),
+                multitransp(np.linalg.solve(point, tangent_vector_b)),
+                axes=point.ndim,
+            )
```

Comment:
This removes a (potential) performance optimization for calls to `norm` where the linear system needs to be solved twice for the same inputs.